### PR TITLE
chore(Windows/Unix): Remove toolbar on systems other than osx

### PR DESCRIPTION
**Pull request by InfamousVague**
### What this PR does 📖

On OSX we NEED to provide a menu bar (for edit->copy paste) etc otherwise the keyboard commands will not work, this is not required in windows or linux so this PR disables it.

### CODE REVIEW

**Review by phillsatellite — DISMISSED** [review]

**Review by tooshel — DISMISSED** [review]
I didn't try on Mac but on Windows it looks good! And you were right Matt, it does look WAY WAY better though I wish it was easer to grab the toolbar to move the window around.

**Comment by phillsatellite** [comment]
Works well on Windows but getting an error when trying to compile on MacOS

image

**Review by phillsatellite — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ cargo run --bin ui --profile=rapid
 **Windows 10+**
 | Dep  | Install Command                                                  |
 |------|------------------------------------------------------------------|
-| Chocolatey | [Installation Guide](https://chocolatey.org/install) |
-| Rust | choco install rust |
+| Rust | [Installation Guide](https://www.rust-lang.org/tools/install) |
+| Protoc | [Download](https://github.com/protocolbuffers/protobuf/releases/download/v22.0/protoc-22.0-win64.zip) |
```
Comment (dariusc93) [inline README.md:52]: May be better to use chocolatey for protoc on windows to avoid alot of the annoyance. Nonblocking to this PR though

**Review by dariusc93 — DISMISSED** [review]
LGTM. Left a comment but nonblocking

**Review by dariusc93 — APPROVED** [review]

**Pull request merged**